internal/executor/flux/jsonschema.json

```diff
@@ -0,0 +1,36 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Flux",
+  "description": "Run the Flux CLI commands directly from your favorite communication platform.",
+  "type": "object",
+  "uiSchema": {
+    "github": {
+      "auth": {
+        "accessToken": {
+          "ui:widget": "password",
+          "ui:help": "Hint: Make it strong!"
```

Comment:
```suggestion
```

Reply:
remember about removing the comma after removing the line 👍